File: Java/p12279.java
import java.util.Scanner;

public class p12279 {
    public static void main(String args[]) {
        Scanner input = new Scanner(System.in);
        int kasus = 1;
        while(true) {
            int byk = input.nextInt();
            if (byk == 0) break;
            int a = 0, b = 0, temp;
            for (int i = 0; i < byk; i++) {
                temp = input.nextInt();
                if (temp == 0) b++;
                else a++;
            }
            
            System.out.printf("Case %d: %d\n", kasus++, a-b);
        }
    }
}

File: Python/p12279.py
kasus = 1
while True:
    byk = int(input())
    if byk == 0:
        break
    masukan = input().split()
    a = 0
    b = 0
    for i in range(0,byk):
        temp = int(masukan[i])
        if temp == 0:
            b+=1
        else:
            a+=1
    
    print("Case {:d}: {:d}".format(kasus, (a-b)))
    kasus+=1